src/Exercise3.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Button = (props) => (
+  <div className="flex justify-center items-center mt-16">
+    <button
```

Comment:
Great Job, but I suggested Button  component to be in different file for good practice.

Reply:
As this component is only used in this file and for this exercise, I think it is better to only have it here

Reply:
Okay great!